### DISCUSSION TITLE
Curriculum development

### DISCUSSION THREAD

**Discussion by Aguncan** [SORTEE-Github-Hackathon/manuscript#183] [Ideas]
Curriculum development is an important step for creating and improving a course or lecture at school or university. As an early carrier researcher who has teaching responsibility develop curriculum for the lectures and update in each semester. Therefore, GitHub can be a good platform for curriculum development. Lecturers and students can easily check the evolution of the lecture through out the semester and I think this is an important for education. 
I am opening this discussion for adding this subject to manuscript as a use of GitHub in Educational materials section. (https://sortee-github-hackathon.github.io/manuscript/v/latest/index.html#educational-materials.)
A quick search from google there were some good examples for curriculum ( e.g. https://github.com/HackYourFuture/curriculum).

**Comment by DrMattG** [comment]
I think this needs to be included in the ms - it would be good to have this link to HackYourFuture and perhaps if we can find one a concrete worked example. I do not do any (well I do 2 weeks in a year!) teaching but I can see that this would be extremely important to highlight in the ms for our target auidence.
